nuxeo-core-binarymanager-s3/README.md

```diff
@@ -75,6 +75,20 @@ by S3 itself. To activate-this mode, use:
 
 See http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html for more.
 
+# Enable CloudFront Direct Download
+
+First, create a CloudFront distribution domain bound to your S3 repository. Doing this allow anyone to access your objects, so we have to enable the `restriction viewer access`, then each generated URL must be signed to allow access.
+
+## Mandatory Parameters
+All S3 parameters (except `nuxeo.core.binarymanager`) are still mandatory; and will be used during the Blob storing process. We need some other parameters:
+
+ - `nuxeo.core.binarymanager=org.nuxeo.ecm.core.storage.sql.CloudFrontBinaryManager`
+ - `nuxeo.s3storage.cloudfront.privKey`: the absolute path of the private key file (`.pem` or `.der`). Read: [Creating CloudFront Key Pairs for Your Trusted Signers](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs) 
+ - `nuxeo.s3storage.cloudfront.privKeyId`: the private key id. 
+ - `nuxeo.s3storage.cloudfront.distribDomain`: the distribution domain name.
+ - `nuxeo.s3storage.cloudfront.protocol`: the prefered protocol (default `HTTPS`)
+ - `nuxeo.s3storage.cloudfront.fix.encoding`: Enable a workaround to fix an error on CloudFront side (default `false`)
```

Comment:
I really would like to link to something; but it is related to a internal support ticket.
And, it is only a temporary flag until they fix the bug on their side.